build.cake

```diff
@@ -168,13 +190,21 @@ Task("Pack-Android")
         NuGetPack(nuGetPackSettings);
     });
 
+Task("Build-Android-Apk")
+    .Does(() =>
+    {
+        BuildAndroidApk(androidProject, true, "Release", settings =>
+            settings
+                .WithProperty("OutputPath", adroidApkOutputDirectory));
+    });
+
 Task("Build-iOS")
     .IsDependentOn("Restore-Packages")
     .Does (() =>
     {
         MSBuild(iOSProject, settings => 
             settings
-                .SetConfiguration(configuration)   
+                .SetConfiguration("Debug")
```

Comment:
Did you mean to commit this change? What happens when you want to pack with config = release?

Reply:
Should've added that instead of changing.  I'll update.